src/transport.cpp

```diff
@@ -1143,6 +1085,7 @@ namespace quicr {
                     handler->subscribe_track_metrics_.bytes_received += obj.payload.size();
                     handler->ObjectReceived({ obj.group_id,
                                               obj.object_id,
+                                              std::nullopt,
```

Comment:
Same comment as before. Starting to get confusing for the application implementer on which object headers are supported based on what they publish that doesn't result in objects being received with those...